### ISSUE TITLE
Delete shortcuts don't work as expected

### ISSUE THREAD

**Issue by bugadani** [lapce/lapce#143]
Hi! Really nice editor, it has a good potential!

I have a few issues with the various delete shortcuts:

When the editor has an active selection, DeleteBackwards deletes the selection and one character before the selection.
DeleteForward only deletes a selection, but does not work when the editor has no text selected.
DeleteForward is not bound to the Del key by default

**Comment by SingingBush** [comment]
This should be re-opened as it's not fixed on master. I've pulled and recompiled moments ago. My _keymaps-linux.toml_ has

```
[[keymaps]]
key = "Delete"
command = "delete_forward"
mode = "i"
```

but when opening keybindings in settings _delete_forward_ was not actually mapped. After I mapped it via the UI I was able to select text and remove it using the Delete key but without first highlighting text the Delete key is still not removing anything.
